io/io/inc/TFile.h

```diff
@@ -25,16 +25,21 @@
 
 #include "Compression.h"
 #include "TDirectoryFile.h"
-#include "TMap.h"
 #include "TUrl.h"
 #include "ROOT/RConcurrentHashColl.hxx"
 
+// not a part of TFile interface, but keep here for a while
```

Comment:
That seems to contradict the fwd decl and the removed `#include`?

Reply:
According to @pcanal suggestion, I just remove `TMap.h` include
But while it is very central place in ROOT, I keep "reminder" to see that was changed.
Most easy way to find such places - search for `R__LESS_INCLUDES` location in the code

Reply:
O, so "it" in "keep here for a while" isn't the `#include` but the comment? That wasn't obvious at all! I suggest to reword:
```suggestion
// Not a part of TFile interface; provide a forward declaration instead of #include.
```

Reply:
I rebase code instead of adding new commits